pkg/driver/device_manager.go

```diff
@@ -34,10 +34,12 @@ type DeviceManager interface {
 	MaxMetricSize() int
 	// MaxRequestsSize return the default maximum request logs size in bytes for this device manager
 	MaxRequestsSize() int
+	// MaxAppLogsSize return the default maximum app logs size in bytes for this device manager
+	MaxAppLogsSize() int
 	// Database safe-to-print (without credentials) path to database
 	Database() string
 	// Init initialize the datastore. If the given database URL is invalid for this type of manager, return false. Return error for actual failures
-	Init(string, int, int, int, int) (bool, error)
+	Init(string, *common.MaxSizes) (bool, error)
```

Comment:
Ah, you already got ahead and put it in a struct? Thanks, this is much better.

Reply:
I think you should pass this by value `common.MaxSizes` and not by reference `*common.MaxSizes`. If you pass it by value, it implies it is changeable (which is shouldn't be). More importantly, you will need to check for `== nil` on every implementation of `Init()`, whereas if you pass it by value, you don't need to do that.